adwords_client/client.py

```diff
@@ -389,10 +375,7 @@ def get_accounts(self, client_id=None):
         return {account['name']: account for account in mcs.get_customers(client_id)}
 
     def get_entities(self, get_internal_operation):
-        try:
-            service_name = self._get_service_from_object_type(get_internal_operation)
-            service = self.service(service_name)
-            results = service.cs_get(get_internal_operation)
-            return [result for result in results]
-        except KeyError:
-            logger.debug('There is no custom service class for this object_type: %s', str(get_internal_operation['object_type']))
+        service_name = self._get_service_from_object_type(get_internal_operation)
+        service = self.service(service_name)
+        results = service.custom_get(get_internal_operation)
+        return list(results)
```

Comment:
👍 : `list(results)`

Se for um iterável.
Se vc tiver certeza que é uma lista, daí retorna direto.